extension/goplugin/schemas.go

```diff
@@ -683,6 +683,20 @@ func (schema *Schema) RegisterType(typeValue interface{}) {
 	schema.env.RegisterType(schema.raw.ID, typeValue)
 }
 
+// Children returns list of schemas that extend schema with given id
+func (schema *Schema) Children() []goext.ISchema {
+	manager := gohan_schema.GetManager()
+	children := []goext.ISchema{}
+	for _, raw := range manager.OrderedSchemas() {
+		for _, parent := range raw.Extends {
+			if parent == schema.ID() {
+				children = append(children, NewSchema(schema.env, raw))
```

Comment:
once you've appended a child to the result list, there's not need to further iterate raw.Extends. add a `brake`